interview-questions.md

```diff
@@ -4,17 +4,19 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: Its used to call back to parent objects, and is easy to recognize.
```

Comment:
Cool! But the interviewer is asking you "Why you would use one", it would benefit to go into detail more than 'easy to recognize'